### PR TITLE
Task 7 formik with hooks

### PR DESCRIPTION
**Pull request by dhaneeshkmrt**

### CODE REVIEW

File: src/invision/components/search-bar/search-bar.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default function SearchBar(props) {
           </div>
         </div>
       </section>
-      <MoviePopup visible={addMoviePopupVisibility} onModalClose={onModalClose} title="Add Movie" />
+      { addMoviePopupVisibility ? <MoviePopup visible={addMoviePopupVisibility} onModalClose={onModalClose} title="Add Movie" /> : null}
```
Comment (shuspes) [inline src/invision/components/search-bar/search-bar.js]: Do you need **visible** property of the **MoviePopup** there?

File: src/invision/components/movie-popup/movie.popup.js [hunk 2]
```diff
@@ -1,89 +1,124 @@
...
+
+      if (!values.runTime) {
+        errors.runTime = 'required';
+      } else if (!isFloat(values.runTime)) {
+        errors.runTime = 'Runtime should be a number';
+      }
+
+      if (!values.rating) {
+        errors.rating = 'required';
+      } else if (!isFloat(values.rating)) {
```
Comment (shuspes) [inline src/invision/components/movie-popup/movie.popup.js:52]: "vote_average" must be less than or equal to 100. I've gotten this error when I've tried to create movie with "2323" rating. You should add this rule to these validation list

**Review by shuspes — APPROVED** [review]

File: src/invision/components/search-bar/search-bar.js (hunk 1 above)
Comment (dhaneeshkmrt) [reply src/invision/components/search-bar/search-bar.js]: removed

File: src/invision/components/movie-popup/movie.popup.js (hunk 2 above)
Comment (dhaneeshkmrt) [reply src/invision/components/movie-popup/movie.popup.js:52]: fixed

**Pull request merged**